README.md

```diff
@@ -1,4 +1,4 @@
-# Large Events
+# Large Events (Knative Proof of Portability)
 
 [![Build Status](https://travis-ci.com/knative-portability/large-events.svg?branch=master)](https://travis-ci.com/knative-portability/large-events)
 [![Coverage Status](https://coveralls.io/repos/github/knative-portability/large-events/badge.svg?branch=master)](https://coveralls.io/github/knative-portability/large-events?branch=master)
```

Comment:
I've noticed this for a while but the coverage button in the readme constantly says 0% even though the one of the coveralls website is correct. Any idea what's causing this or should we make a bug PR?

Reply:
The image that is loaded is cached in your browser. [Navigate to the repo](https://github.com/knative-portability/large-events), then ctrl+F5.

Reply:
Didn't know about that, thanks!